python_scripts/one_time_scripts/refid.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+# This script updates caas_aspace_refid's based on a provided CSV.  For help on available arguments and options:
+# `python repeatable/update_fileuri.py -h`.
+import argparse
+import os
+import sys
+
+from dotenv import load_dotenv, find_dotenv
+from loguru import logger
+from pathlib import Path
+from python_scripts.utilities import client_login, read_csv
+
+# Logging
+logger.remove()
+log_path = Path(f'./logs', 'update_refid_{time:YYYY-MM-DD}.log')
+logger.add(str(log_path), format="{time}-{level}: {message}")
+
+# Find  and load environment-specific .env file
+env_file = find_dotenv(f'.env.{os.getenv("ENV", "dev")}')
+load_dotenv(env_file)
+
+def parseArguments():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("csvPath", help="path to csv input file", type=str)
+    parser.add_argument("-dR", "--dry-run", help="dry run?", action='store_true')
+    parser.add_argument("--version", action="version", version='%(prog)s - Version 1.0')
+
+    return parser.parse_args()
+    
+def get_refid(client, resource_uri):
```

Comment:
If we have to run it again, we could think about adding docstrings to these functions - but they are pretty readable and understandable as is.